packages/app/src/runner/selector-playground/SelectorPlayground.spec.tsx

```diff
@@ -4,7 +4,7 @@ import SelectorPlayground from './SelectorPlayground.vue'
 
 // TODO: Test is failing due to Percy conflict
 // eslint-disable-next-line
-describe.skip('SelectorPlayground', () => {
+describe('SelectorPlayground', () => {
```

Comment:
Are we able to remove the `// TODO: Test is failing due to Percy conflict`?

Reply:
I also think the `// eslint-disable-next-line` above is no longer needed. Linter doesn't seem to be complaining 😄.

Reply:
Updated here: [`b9d1636` (#19413)](https://github.com/cypress-io/cypress/pull/19413/commits/b9d163621ee8442545e703d273b98f2a8c33c5b3)